libs/agno/agno/session/workflow.py

```diff
@@ -75,6 +75,47 @@ def upsert_run(self, run: WorkflowRunOutput) -> None:
         else:
             self.runs.append(run)
 
+    def get_workflow_history(self, session: WorkflowSession, num_history_runs: int = 3) -> Optional[str]:
```

Comment:
```suggestion
    def get_workflow_history(self, session: WorkflowSession, num_runs: int = 3) -> Optional[str]:
```

Reply:
Why do you pass a session? Why not just use `self`?

Reply:
I would split this into a `get_workflow_history` function that returns a list of tuples (or something like that), and another function `get_workflow_history_context` that returns the str (and it calls the get_workflow_history function)

That gives users more options.

Reply:
makes sense, updating!